integration-testing/src/test/java/org/triplea/server/http/BasicEndpointTest.java

```diff
@@ -23,8 +23,10 @@ protected void verifyEndpointReturningVoid(final Consumer<T> methodRunner) {
   }
 
   /** Use this to verify an endpoint that returns data. */
-  protected void verifyEndpointReturningObject(final Function<T, ?> methodRunner) {
+  protected <X> X verifyEndpointReturningObject(final Function<T, X> methodRunner) {
+    final X result = methodRunner.apply(clientBuilder.apply(localhost));
     assertThat(methodRunner.apply(clientBuilder.apply(localhost)), notNullValue());
```

Comment:
```suggestion
    assertThat(result, notNullValue());
```
The space difference at the beginning of the diff seems to be just a visual bug, no space is actually added